sonar-text-plugin/src/main/java/org/sonar/plugins/secrets/SecretsRulesDefinition.java

```diff
@@ -49,15 +48,14 @@ public DefaultQualityProfile() {
   }
 
   public static List<Class<?>> checks() {
-    List<Class<?>> checkList = new ArrayList<>(GeneratedCheckList.generatedChecks());
```

Comment:
Do we still plan to use `GeneratedCheckList`?

Reply:
I don't believe so. Should be ideated after the sprint.